### PR TITLE
[MOD-121][Improvement] Add html templates for moderation notifications

### PR DESCRIPTION
**Pull request by laurenbarker**
## Purpose

screen shot 2017-10-04 at 12 35 10 pm

#### Rendered emails:
https://jsfiddle.net/g6m2cymc/
https://jsfiddle.net/e6xdxm8c/
https://jsfiddle.net/y6a7gubh/

## Ticket

https://openscience.atlassian.net/browse/MOD-121

### CODE REVIEW

**Comment by sheriefvt** [comment]
you can replace txt with html in this line https://github.com/CenterForOpenScience/osf.io/blob/fbd43d12c55c226e55ea77268aca16202ad071c7/tests/test_notifications.py#L1840 for the failing test

**Comment by laurenbarker** [comment]
Thanks @sheriefvt!

File: website/templates/emails/reviews_submission_confirmation.html.mako [hunk 1]
```diff
@@ -0,0 +1,36 @@
+## -*- coding: utf-8 -*-
+<%inherit file="notify_base.mako"/>
+<%def name="content()">
+    <div style="margin: 40px;">
+        Hello ${user.fullname},
+        <br><br>
```
Comment (aaxelb) [inline website/templates/emails/reviews_submission_confirmation.html.mako]: Why so chilly? `<p>` seems more appropriate

File: website/templates/emails/reviews_submission_confirmation.html.mako [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        For more information about ${reviewable.provider.name}, visit <a href="${provider_url}">${provider_url}</a> to learn more. To learn about the Open Science Framework, visit <a href="https://osf.io/">https://osf.io/</a>.
+        <br><br>
+        For questions regarding submission criteria, please email ${provider_contact_email}
+        <br><br>
+        <br>
+        Sincerely,<br>
+        Your ${reviewable.provider.name} and OSF teams
+        <br><br>
+        Center for Open Science<br>
```
Comment (aaxelb) [inline website/templates/emails/reviews_submission_confirmation.html.mako]: Except here, this `<br>` is good.

File: website/templates/emails/reviews_submission_confirmation.html.mako (hunk 1 above)
Comment (laurenbarker) [reply website/templates/emails/reviews_submission_confirmation.html.mako]: I agree, I was just copying the other templates.

**Pull request merged**